### DISCUSSION TITLE
Render Doc 1.25

### DISCUSSION THREAD

**Discussion by Negus72** [eliemichel/MapsModelsImporter#273] [Q&A]
Hi all. How can i connect Render Doc 1,25 (no injection) with Chrome?
Thanks in advance

**Comment by WingmanMedia** [comment]
I have been using the version 1.13 supplied with blender 3.0 and  that works. I tried to update RenderDoc 1.13 to the last version a couple of months ago and everything stopped working. I had to roll back to make it working again.
